internal/config/secrets.go

```diff
@@ -60,7 +60,7 @@ func loadSecrets(config *load.Config) {
 				load.Logrus.WithFields(logrus.Fields{
 					"secret": name,
 					"kind":   secret.Kind,
-				}).Error("config: secret missing region")
+				}).Error("config: secret missing: 'region'")
```

Comment:
I would use the same format as below: `.. requires 'region' parameter to be set`